### PR TITLE
Implementation

### PR DESCRIPTION
**Pull request by sophiewigmore**
## Summary

Implements #4 

Buildpack creates the httpd.conf file and makes it available at
$PHP_HTTPD_PATH at launch-time
~~Blocked on https://github.com/paketo-buildpacks/php/issues/503 / associated RFC.~~

## Checklist

* [X] I have viewed, signed, and submitted the Contributor License Agreement.
* [X] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [X] I have added an integration test, if necessary.
* [X] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [X] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Comment by sophiewigmore** [comment]
@paketo-buildpacks/php-maintainers ready for review now RFC is. merged

**Comment by sophiewigmore** [comment]
waiting for https://github.com/paketo-buildpacks/rfcs/pull/162

File: build.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+// ConfigWriter sets up the HTTPD configuration file from via Go templating and user-set environment variables.
+type ConfigWriter interface {
+	Write(layerPath, workingDir, cnbPath string) (string, error)
+}
+
+// Build will return a packit.BuildFunc that will be invoked during the build
+// phase of the buildpack lifecycle.
+//
+// Build will create a layer dedicated to PHP FPM configuration, configure default FPM
```
Comment (thitch97) [inline build.go:29]: Should this be `HTTPD` instead of `PHP FPM`?

File: build_test.go [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+							{Name: phphttpd.PhpHttpdConfigLayer},
+						},
+					},
+					Layers: packit.Layers{Path: layerDir},
+				})
+				Expect(err).To(MatchError(ContainSubstring("config writing error")))
+			})
+		})
+
+		context("when config file cannot be written", func() {
```
Comment (thitch97) [inline build_test.go:196]: Duplicate case?

**Review by thitch97 — APPROVED** [review]

**Pull request merged**